Tasks/AzureResourceManagerTemplateDeploymentV3/operations/ManagementGroup.ts

```diff
@@ -0,0 +1,17 @@
+import deployAzureRG = require("../models/DeployAzureRG");
+import armResource = require("azure-arm-rest-v2/azure-arm-management-group");
+import { DeploymentScopeBase } from "./DeploymentScopeBase";
+import { DeploymentParameters } from "./DeploymentParameters";
+
+export class ManagementGroup extends DeploymentScopeBase {
```

Comment:
This class does not look to be really a required one. from main.ts, we can directly call base class constructor and pass ManagementGroupClient object

Reply:
Correct